### DISCUSSION TITLE
Welcome to FiraCode Discussions!

### DISCUSSION THREAD

**Discussion by tonsky** [tonsky/FiraCode#1166] [General]
Let’s try out this new Discussions thing!

Here’s what it is about:

* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together.

Feel free to comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RubenVerg** [comment]
Err... I'm Ruben, student, I mainly code TypeScript and I like to suggest ligatures and discuss others'

**Reply by tonsky** [reply]
Hi Ruben! Thank you for breaking the ice :) I like your ideas, please keep them coming!

**Reply by RubenVerg** [reply]
I mean, I did expect people to already have written. I sent by email, yk

**Comment by tonsky** [comment]
I am Nikita, author of Fira Code. I built it for myself out of love for Fira Mono and Hasklig ligatures, happy that people found it useful. I am programmer by trade, and spend most of my days in Sublime Text, Terminal or IntelliJ Idea.

**Comment by wiznillyp** [comment]
This font does work with MATLAB, at least in Windows. MATLAB uses a JRE, so you have to load the TTFs into the MATLAB directory as described in this [link](https://www.mathworks.com/matlabcentral/answers/514119-can-i-add-custom-fonts-to-my-matlab-desktop-in-preferences).

In short, load the font into: `MATLAB\<ver>\sys\java\jre\win64\jre\lib\fonts` then restart MATLAB.

I have tested this with Windows 10, MATLAB2020a and it works incredibly.  Thanks!

**Reply by tonsky** [reply]
Can you write a wiki entry on that, please? And PR adding it to compatible talbe, with link to the wiki instructions. Thank you!

**Reply by wiznillyp** [reply]
All set, I think.  Thanks for the gorgeous font.
